samples/cellular/modem_shell/sample.yaml

```diff
@@ -95,8 +95,8 @@ tests:
       - nrf9151dk/nrf9151/ns
       - nrf9160dk/nrf9160/ns
       - nrf9161dk/nrf9161/ns
-    extra_args: SHIELD=nrf7002ek EXTRA_CONF_FILE=overlay-nrf700x-wifi-scan-only.conf
-                SB_CONFIG_WIFI_NRF700X=y SB_CONFIG_WIFI_NRF700X_SCAN_ONLY=y
+    extra_args: SHIELD=nrf7002ek_nrf7000 EXTRA_CONF_FILE=overlay-nrf700x-wifi-scan-only.conf
+                SB_CONFIG_WIFI_NRF70=y SB_CONFIG_WIFI_NRF70_SCAN_ONLY=y
     tags: ci_build sysbuild ci_samples_cellular
   sample.cellular.modem_shell.nrf7000ek_wifi:
```

Comment:
`sample.cellular.modem_shell.nrf7001ek_wifi` was removed. This `sample.cellular.modem_shell.nrf7000ek_wifi` looks to me exactly the same as fixed `sample.cellular.modem_shell.nrf7002ek_wifi` configuration so should we remove `sample.cellular.modem_shell.nrf7000ek_wifi`.
@krish2718

Reply:
I have used nRF7000 for scan only everywhere and nRF7002/7001 for full Wi-Fi connection.